.github/workflows/full_stack_tests.yml

```diff
@@ -263,11 +271,11 @@ jobs:
       
       - name: Upload desktop test recordings as an artifact
         # We are checking for failure of desktop-test-attempt-1 and not desktop-test-attempt-2 because
-        # because we want to upload video recording regardless of whether desktop-test-attempt-2 passes or fails.
+        # we want to upload video recording regardless of whether desktop-test-attempt-2 passes or fails.
         if: ${{ steps.desktop-test-attempt-1.outcome == 'failure' }}
         uses: actions/upload-artifact@v4
         with:
-          name: desktop-acceptance-test-video-${{steps.generate_suite_name_for_artifacts.outputs.MODIFIED_SUITE_NAME}}
+          name: desktop-acceptance-test-video-attempt-2-${{steps.generate_suite_name_for_artifacts.outputs.MODIFIED_SUITE_NAME}}
```

Comment:
Per [this comment](https://github.com/oppia/oppia/pull/23166#discussion_r2293110664), isn't this supposed to be attempt-1, and shouldn't the upload be done after the first test attempt rather than the second?